credit_card_gateway.go

```diff
@@ -57,3 +62,30 @@ func (g *CreditCardGateway) Delete(ctx context.Context, card *CreditCard) error
 	}
 	return &invalidResponseError{resp}
 }
+
+//  ExpiringBetween Find list of credit card that expire between the specified dates
+func (g *CreditCardGateway) ExpiringBetween(ctx context.Context, fromDate, toDate time.Time) ([]*CreditCard, error) {
+	path := fmt.Sprintf("/payment_methods/all/expiring?start=%s&end=%s",
+		fromDate.Format("012006"),
+		toDate.Format("012006"))
```

Comment:
According to the braintree/braintree_java SDK we should send these dates in UTC.
Ref: [braintree/braintree_java](https://github.com/braintree/braintree_java/blob/2def3f6/src/main/java/com/braintreegateway/CreditCardGateway.java#L211)